main.py

```diff
@@ -3236,6 +3236,8 @@ async def claim_reward(user, channeley, type):
     except Exception:
         pass
 
+    if randint(0, 10000) == 236:
```

Comment:
```suggestion
    if random.randint(0, 10000) == 236:
```

Reply:
i thought randint was already imported like `from random import randint` :staring_cat:

Reply:
it was until some commits ago

Reply:
im going to airstrike your house from orbit

Reply:
agreed